README.md

```diff
@@ -1,2 +1,2 @@
-Alpha version of the ROS2 Universal Robots driver. Should be transferred to the Universal Robots org when ready.
```

Comment:
@skolk why was the note about transferrance removed? We do plan on doing this, because it makes sense for the HW vendor to support their drivers long term. We don't have the resources to support it ourselves. Also, this is what was agreed to in the initial meetings with all the stakeholders.

Reply:
https://github.com/PickNikRobotics/Universal_Robots_ROS2_Driver/pull/59